src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -121,9 +121,10 @@
  * ///////////////////////////////////////////////////
  * package com.whitespace; // violation, 'package' should be separated from previous line.
  * import java.io.Serializable; // violation, 'import' should be separated from previous line.
- * class Foo { // violation, 'CLASS_DEF' should be separated from previous line.
- *   public static final int FOO_CONST = 1;
- *   public void foo() {} // violation, 'METHOD_DEF' should be separated from previous line.
+ * class FirstClass { // violation, 'CLASS_DEF' should be separated from previous line.
+ *   public int firstClassVariable = 1;
+ *   public void classMethod() {} // violation, 'METHOD_DEF'
+ *                                // should be separated from previous line.
```

Comment:
Shorten name of variable to make single line as to was before.

Reply:
Done. Variable length has been shortened.